lighthouse-core/audits/autocomplete.js

```diff
@@ -25,10 +25,161 @@ const UIStrings = {
    'effort, consider enabling autocomplete by setting the `autocomplete` ' +
    'attribute to a valid value.' +
   ' [Learn more](https://developers.google.com/web/fundamentals/design-and-ux/input/forms#use_metadata_to_enable_auto-complete)',
+  /** Label for a column in a data table; entries will be the autocomplete token suggestions based on heuristics. */
+  columnAutocompleteSuggestions: 'Autocomplete Suggested Token',
```

Comment:
Just "Suggested value" and "Current value". We don't have a lot of horizontal space.

Reply:
in my earlier review i suggested "token" wording for the suggested case. in the case of `<input autocomplete="section-foo shipping" class="addressln1">`, right now we're putting `address-line1` in the suggested column. (no mention of the two optional tokens: `section-foo shipping`). 

we _could_ include those two in the column value, but it'd be pretty awkward.

Reply:
So it'd show current = `section-foo shipping`, suggested = `address-line1`? That seems more awkward.

Reply:
yeah i was saying awkward to code, but awkward in the UX is true. 
"Current Value" = `section-foo shipping` and "Suggested **Token**" = `address-line1`
of course this is not too realistic because it's weird to use autocomplete hints, but  _just_ the optional tokens.. 😛  slightly more realistic is a typo on the primary token.

and **much more** realistic is...  there's there billing and shipping address fields on one page. no inputs have autocomplete hints.

we can suggest "address-line1", etc, but in reality they will also want to apply "shipping"/"billing" in front of them. 

also all telephone numbers.. for those we'll say suggested = "tel", but they almost _have_ to add home/work/mobile in front of that, for it to be useful

Reply:
Does the chrome prediction provide just a single token or does it suggest what the entire thing should be?

Reply:
just the single "primary" token.

Reply:
> just the single "primary" token.

in the spec they call the "primary token" the "autofill field name".. i personally like "primary token" but that's my term.

the first 3 sections are "optional tokens"

`[section-*] [shipping|billing] [home|work|mobile|fax|pager] <autofill field name>`

> just the single "primary" token.

OK looking again, the prediction data mighttt actually have some hints on if something is considered shipping/billing or mobile/home/work.. interesting.

![image](https://user-images.githubusercontent.com/39191/91749718-c743d080-eb76-11ea-8f28-d0fa0ac7a077.png)

but i don't think we can open that door right now. at least not this week.

Reply:
![Screen Shot 2020-08-31 at 8 27 11 PM](https://user-images.githubusercontent.com/34068912/91781457-5d541700-ebc8-11ea-91f0-ae41b57cf353.png)

Reply:
also good to note though that it has the billing section as shipping as well @paulirish

Reply:
Can you remove `Autocomplete` from both of these column header strings?